pandas/tests/series/methods/test_size.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from pandas import Series
+
+
+@pytest.mark.parametrize(
+    "data, index, expected",
+    [
+        ([1, 2, 3], None, 3),
+        ({"a": 1, "b": 2, "c": 3}, None, 3),
+        ([1, 2, 3], ["x", "y", "z"], 3),
+        ([1, 2, 3, 4, 5], ["x", "y", "z", "w", "n"], 5),
+        ([1, 2, 3], None, 3),
+        ([1, 2, 3], ["x", "y", "z"], 3),
+        ([1, 2, 3, 4], ["x", "y", "z", "w"], 4),
+    ],
+)
+
+# GH#52897
+def test_series(data, index, expected):
+    ser = Series(data, index=index)
```

Comment:
Ah, sorry - the comment should be the first line of the test. Nothing wrong with this really, but I do think we should be consistent with the style of the other tests.

```
def test_series(data, index, expected):
    # GH#52897
    ser = Series(data, index=index)
```

Same goes for the other test as well.

Reply:
Thanks for the note! All changes should be pushed.